setup.py

```diff
@@ -46,20 +46,16 @@
 f.close()
 
 portrait_files = ["src/python/graphics/share/portraits.scr", ]
-cmip5_amip_json = glob.glob("data/CMIP_metrics_results/CMIP5/amip/*.json")
-cmip5_historical_json = glob.glob(
-    "data/CMIP_metrics_results/CMIP5/historical/*.json")
-cmip5_piControl_json = glob.glob(
-    "data/CMIP_metrics_results/CMIP5/piControl/*.json")
-demo_ACME_files = glob.glob("demo/ACME/*.py")
-demo_CSIRO_files = glob.glob("demo/CSIRO/*.py")
-demo_CSIRO_files += glob.glob("demo/CSIRO/*.png")
-demo_CSIRO_files += glob.glob("demo/CSIRO/*.sh")
-demo_GFDL_files = glob.glob("demo/GFDL/*.py")
-demo_GFDL_files += glob.glob("demo/GFDL/*.png")
-demo_NCAR_files = glob.glob("demo/NCAR/*.py")
-param_files = glob.glob("doc/parameter_files/*.py")
-
+demo_ACME_files = glob.glob("demo/external-setups/ACME/*.py")
```

Comment:
and why do we need to package these files? These are examples/docs we can link to these from some wiki. If yuor demo does not use it, then it shouldn't be packaged. No-one will look at it buried inside the python distribution.